bugsnag-android-core/build.gradle

```diff
@@ -11,6 +11,10 @@ bugsnagBuildOptions {
 apply plugin: "com.android.library"
 apply plugin: "org.jetbrains.dokka"
 
+dependencies {
+    implementation 'com.dslplatform:dsl-json:1.9.8'
```

Comment:
I used this benchmark as a guide: https://github.com/fabienrenaud/java-json-benchmark

Most Java-based JSON packages are very slow because they convert to strings, which then have to be converted to UTF-8. I chose this one because it has the fastest performance, and it also can read/write directly from byte arrays (which we need for the native-java shared memory region).